src/IROperator.h

```diff
@@ -307,6 +307,16 @@ Expr remove_likelies(const Expr &e);
  * all calls to likely() and likely_if_innermost() removed. */
 Stmt remove_likelies(const Stmt &s);
 
+/** If the expression is a tag helper call, remove it and return
```

Comment:
remove_likelies is recursive (i.e. it's an IRMutator), but strip tags is only shallowly recursive. This seems like a potential source of error. Should strip_tags just be a full mutator?

Reply:
This helper is used in place of code that used to do things like:
```
if (const Call *c = as_likely(x)) {
  x = c->args[0];
}
...
```

Where x could be used recursively where the more deeply nested tags might be needed, so I think `strip_tags` needs to have a different meaning than `remove_likelies`.

I have only seen a use case for recursively removing all tags for a specific tag: there's `remove_likelies`, and there is a one-off mutator to remove `predicate`, but these are different than the use cases for `strip_tags`.

Reply:
(sorry, messages crossed there.)

What do you think of `unwrap_tags`?